runtimes/go-1.19/build.sh

```diff
@@ -0,0 +1,24 @@
+#/bin/sh 
+
+#Fail bild if any command fails
+set -e 
+
+# Preapre separate directory to prevent changing user's files
+cp -R /usr/code/* /usr/builds
+# Get server.go(temporarily) file to build go.mod file
+cp /usr/local/src/server/server.go /usr/builds 
+
+# Install User Function dependencies by creting a mod file which holds both user 
+# and our server dependencies
+cd /usr/builds
+go mod init openruntime/server  
+go mod tidy 
+go mod vendor
+go clean -modcache
+
+# Remove server.go file, as we copy all these files to server folder
+# again for starting the server
+rm server.go 
+
```

Comment:
This works, but I think it might be better to actually compile the code like how we do with [dart](https://github.com/open-runtimes/open-runtimes/blob/0.1.3/runtimes/dart-2.17/build.sh#L37).

Reply:
Got it.